trestle/oscal/catalog.py

```diff
@@ -70,182 +44,36 @@ class Control(OscalBaseModel):
+        description=
+        'Identifies a control such that it can be referenced in the defining catalog and other OSCAL instances (e.g., profiles).',
+        title='Control Identifier'
+    )
+    class_: Optional[constr(
+        regex=
+        r'^[_A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD][_A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD\-\.0-9\u00B7\u0300-\u036F\u203F-\u2040]*$'
+    )] = Field(
         None,
         alias='class',
         description='A textual label that provides a sub-type or characterization of the control.',
-        title='Control Class',
+        title='Control Class'
     )
     title: str = Field(
         ...,
         description='A name given to the control, which may be used by a tool for display and navigation.',
-        title='Control Title',
-    )
-    params: Optional[List[common.Parameter]] = Field(
-        None
+        title='Control Title'
     )
+    params: Optional[List[common.Parameter]] = Field(None)
     props: Optional[List[common.Property]] = Field(None)
     links: Optional[List[common.Link]] = Field(None)
     parts: Optional[List[common.Part]] = Field(None)
     controls: Optional[List[Control]] = None
 
 
-class Address(OscalBaseModel):
-    """
-    A postal address for the location.
-    """
-
-    class Config:
-        extra = Extra.forbid
-
-    type: Optional[Union[constr(regex=r'^[_A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD][_A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD\-\.0-9\u00B7\u0300-\u036F\u203F-\u2040]*$'), TypeEnum1]] = Field(
-        None, description='Indicates the type of address.', title='Address Type'
-    )
-    addr_lines: Optional[List[constr(regex=r'^\S(.*\S)?$')]] = Field(
-        None, alias='addr-lines'
-    )
-    city: Optional[constr(regex=r'^\S(.*\S)?$')] = Field(
-        None,
-        description='City, town or geographical region for the mailing address.',
-        title='City',
-    )
-    state: Optional[constr(regex=r'^\S(.*\S)?$')] = Field(
-        None,
-        description='State, province or analogous geographical region for a mailing address.',
-        title='State',
-    )
-    postal_code: Optional[constr(regex=r'^\S(.*\S)?$')] = Field(
-        None,
-        alias='postal-code',
-        description='Postal or ZIP code for mailing address.',
-        title='Postal Code',
-    )
-    country: Optional[constr(regex=r'^\S(.*\S)?$')] = Field(
-        None,
-        description='The ISO 3166-1 alpha-2 country code for the mailing address.',
-        title='Country Code',
-    )
-
-
-class Location(OscalBaseModel):
```

Comment:
I´m assuming that location has been deleted automatically due to moving it to the common class right?

Reply:
Yes.